definitions/npm/cache-manager_v2.x.x/flow_>=v0.28.x/cache-manager_v2.x.x.js

```diff
@@ -0,0 +1,32 @@
+declare module 'cache-manager' {
+  declare type _CacheOptions = {
+    ttl?: number,
+  }
+  declare type CacheOptions = $Shape<_CacheOptions> | Object
```

Comment:
`$Shape<_CacheOptions> | Object` is actually just as permissive/restrictive as just `Object`: "Either an object with this shape, or any object" is the same as "any object".

We don't really have a good way of saying what I think you're trying to say here.